kubernetes/config-maps/nginx/ssl.conf

```diff
@@ -0,0 +1,26 @@
+# Disable SSLv3
+ssl_protocols              TLSv1 TLSv1.1 TLSv1.2;
```

Comment:
Please disable `TLSv1` and `TLSv1.1`.
- TLS 1.0 is vulnerable to multiple attacks and no longer considered secure. Since 2016 the recommendation is to have it disabled: [SSL and TLS 1.0 No Longer Acceptable for PCI Compliance](https://blog.varonis.com/ssl-and-tls-1-0-no-longer-acceptable-for-pci-compliance/)
- TLS 1.1 is still OK but it's never been in widely adopted (see [Deprecating TLS 1.0 & 1.1](https://www.digicert.com/blog/depreciating-tls-1-0-and-1-1/), [Is there a reason for still having TLSv1.1 enabled?](https://community.qualys.com/thread/16565-is-there-a-reason-for-still-having-tlsv11-enabled)). Any client supporting it is very likely to support TLS 1.2 too so there's no reason not to force 1.2.

And in any case, we can afford to offer only TLS 1.2 because we don't have to support ancient browsers (or any browsers at all for that matter).

Reply:
Done.